File: Problem42.java
public class Problem42 {
	private int[] triangles = {1, 3, 6, 10, 15, 21, 28, 36, 45, 55, 66, 78, 91, 05, 20, 36, 53, 71, 190};
	
	public static void main( String[] args ) {
		int triange = 0;
		
		for( int i = 1; i < 20; i++ ) {
			triange = (i * (i + 1)) / 2;
			System.out.println( triange );
		}
	}
}

File: Problem42.py
triangles = []
for i in range( 0, 21 ):
	triangles.append( i * (i + 1) / 2 )

f = open( 'assets/words.txt', 'r' )

content = f.read()
words = content.replace( '"', '' )
words = words.split( ',' )

are = 0
are_not = 0
for word in words:
	word_value = 0
	for letter in word:
		word_value += ord(letter) - 64

	if word_value in triangles:
		are += 1
		print word
	else:
		are_not += 1

print are, are_not